### DISCUSSION TITLE
[general] Redundant arguments in `tar_quarto`?

### DISCUSSION THREAD

**Discussion by svraka** [ropensci/tarchetypes#155] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

It was discussed previously in <https://github.com/ropensci/tarchetypes/discussions/124#discussioncomment-4150921>, how one needs to specify all the packages to load in Quarto documents, because unlike with R Markdown, knitting happens in an R session launched by Quarto, not by targets, thus the list of packages loaded by target cannot be passed on. As far as can tell, this is still not documented anywhere else. Nevertheless, there is a `packages` argument in `tar_quarto` and its docstring simply inherits from `tar_target`, which makes the users believe that packages would be loaded in a Quarto document in a similar way to other types of targets.

Does the `packages` argument still do anything in `tar_quarto`? Either way, I think a note on package loading with Quarto would be useful.

I haven't investigated other arguments in `tar_quarto` but the same issues could come up with other arguments (`library`, `error`, etc.), especially as in the R session launched by Quarto targets is not set up and `tar_option_get` will return default values, not the ones specified in the pipeline.

(BTW, is this the right place to raise an issue? None of the issue templates really fit.)

**Comment by wlandau** [comment]
That makes sense. As of be67cf25fb0296bb1af77fd0734784891106e74e, the `packages` and `library` arguments are deprecated in  `tar_quarto()` and `tar_quarto_raw()`. I kept them in `tar_quarto_rep()` and `tar_quarto_rep_raw()` because the upstream set of parameters is its own target that may need packages.
